frappe/geo/country_info.json

```diff
@@ -2,81 +2,112 @@
  "Afghanistan": {
   "code": "af",
   "currency": "AFN",
-  "currency_fraction": "Pul",
+  "currency_fraction": "Puls",
   "currency_fraction_units": 100,
+  "currency_name": "Afghanis",
   "currency_symbol": "\u060b",
+  "date_format": "dd-mm-yyyy",
   "number_format": "#,###.##",
+  "smallest_currency_fraction_value": 0.01,
+  "time_format": "HH:mm:ss",
```

Comment:
```suggestion
  "currency": {
    "currency_name": "AFN",
    "symbol": "\u060b",
    "unit_name": "Afghanis",
    "fraction_name": "Puls",
    "fraction": "Puls",
    "fraction_units": 100,
    "smallest_fraction_value": 0.01
  }
```

Maybe we can name the keys consistently with the field-names of DocType **Currency**. This would be less confusing and also save the mapping during import.

Reply:
Also, is `fraction_units` and `smallest_fraction_value` not the same information, assuming `smallest_fraction_value == 1 / fraction_units`?

Reply:
This is a vast improvement on what is already there, not a complete refactor/rewrite. If you want to make simplifications afterward, that's fine.

Reply:
Sorry, I know it's tedious to make these changes. Nevertheless I think reviews are crucial for code quality (and UX, in the case above). You cannot rely on other people fixing things afterwards.

Reply:
You are suggesting making chamges that are not backward compatible, which is something everyone should be specifically avoiding unless absolutely necessary, because they don't know the history and what other code might be affected.

Reviewers need to be very careful to comment on **necessary** changes and bug fixes only and not ask volunteers to spend more of their unpaid time to develop additional and unnecessary functionality.

I won't be spending any more time on this, but I'm happy to keep it in my companies' branch for my customers to benefit from.